mappings/ctv3sctmap2/mappers.py

```diff
@@ -88,3 +89,39 @@ def ctv3_to_snomedct(ctv3_ids):
             )
 
     return records
+
+
+def snomedct_to_ctv3(snomedct_ids):
+    """Convert SNOMED CT Concept codes to CTV3 Concepts IDs."""
+    ctv3_ids = set(
+        CTV3Concept.objects.filter(
+            snomedct_mappings__sct_concept__active=True,
```

Comment:
Why are we filtering out inactive SNOMED concepts here?

Reply:
The reverse of this function only looks for active Snomed concepts, should this function ignore that state?

Reply:
When going from SNOMED to CTV3, it makes sense not to exclude inactive SNOMED concepts, since the SNOMED concepts are provided by the caller of the function.

When going from CTV3 to SNOMED, I'm not sure why we've previously ignored inactive SNOMED concepts that are in the mapping.  I'll do some digging through emails/Slack, and make sure this is written up somewhere.

Reply:
Ah, of course, good point!  Fixed in 3833abc

Reply:
Turns out this is moot, because no inactive SNOMED concepts are included in the mappings that are both assured and have `map_status=True`

```
db.sqlite3> select active, count(*) from snomedct_concept where id in (select distinct sct_concept_id from ctv3sctmap2_mapping where is_assured and map_status) group by active;                           
+--------+----------+
| active | count(*) |
+--------+----------+
| 1      | 150725   |
+--------+----------+
```

Further thought required (outside the scope of this PR) about what, if anything, we can do about mapping from inactive SNOMED concepts.